ec/src/hashing/field_hashers/mod.rs

```diff
@@ -51,8 +49,7 @@ impl<F: Field, H: VariableOutput + Update + Sized + Clone> HashToField<F>
 {
     fn new_hash_to_field(domain: &[u8], count: usize) -> Result<Self, HashToCurveError> {
         // Hardcode security parameter
-        let security_parameter = 128;
-        let bytes_per_base_field_elem = hash_len_in_bytes::<F>(security_parameter, count);
+        let bytes_per_base_field_elem = hash_len_in_bytes::<F, 128>(count);
```

Comment:
Note that this won't be compatible with the `pasta_curves` implementation; we use 256.

Here's the argument for why:
* There is no performance loss from doing so, at least when using XMD:BLAKE2b.
* If you're aiming for a 128-bit security level, and being pedantic about provable concrete security, then all of the ~2<sup>-128</sub> success probabilities for the attacker add up and you have to take them into account for the final security bound. Whereas any 2<sup>-256</sub> success probabilities are still completely negligible and don't matter.

Reply:
We are going to add different FieldHasher(s) and work on compatibility with different network on our next iteration. I'd leave it as it was from the original pull request for now to facilitate merging. Unless @Pratyush would prefer to change it for the DefaultFieldHasher.

Reply:
One middle ground would be to expose the security parameter as a const parameter on `DefaultFieldHasher`?

Reply:
@Pratyush Done in https://github.com/w3f/arkworks-algebra/pull/35/commits/15677da9468bba5fe689b017c189197f893280bf